src/config_parse.c

```diff
@@ -351,7 +351,7 @@ static int parse_multiline_variable(git_config_parser *reader, git_buf *value, i
 		}
 
 		/* If it was just a comment, pretend it didn't exist */
-		quote_count = strip_comments(line, !!in_quotes);
+		quote_count = strip_comments(line, in_quotes % 2);
```

Comment:
Is this change even required now? With the below change for how we call `parse_multiline_variable`, were now only ever passing either `0` or `1` to this function, aren't we?

Reply:
Yes, it seems to work without this change.